database/seeds/ProductionUsersTableSeeder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use App\Role;
+
+class ProductionUsersTableSeeder extends Seeder
```

Comment:
Rename the class appropiately for example DefaultUserSeeder

Reply:
The ordinary convention to name this is /* table name*/TableSeeder, but I can't use this convention because it's already used for development seeder, so how should I rename it?